test/SILOptimizer/OSSignpostAnimationBeginTest.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -emit-ir -swift-version 5 -O -primary-file %s | %FileCheck %s --check-prefix=CHECK --check-prefix=CHECK-%target-ptrsize
+//
+// REQUIRES: VENDOR=apple
+//
+// This tests the OSLogOptimization as it runs on os_signpost(.animationBegin, ...) calls. 
+// This only tests that the OSLogOptimization pass does not crash when evaluating an animation
```

Comment:
This needs a sil test case. Can you add one?

Reply:
@guitard0g You can add a SIL test to `OSLogMandatoryOptTest.sil`.